docs/operate/troubleshooting.mdx

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 13
+description: "Resolving common problems with Restate deployments"
+---
+
+import Admonition from '@theme/Admonition';
+import { Terminal } from "../../src/components/code/terminal";
```

Comment:
```suggestion
import { Terminal } from "@site/src/components/code/terminal";
```

to use absolute paths.

Reply:
Nice! I only learned of this trick when I looked at your recent changes, this is much nicer!